libcontainer/cgroups/systemd/systemd_test.go

```diff
@@ -0,0 +1,29 @@
+package systemd
+
+import (
+	"testing"
+)
+
+func TestSystemdVersion(t *testing.T) {
+	var systemdVersionTests = []struct {
+		verStr       string
+		expectedVer  int
+		expectErrNil bool
```

Comment:
I would invert the meaning of this and call it "expectErr". Reads easier this way (especially at 7am:))